src/app/[locale]/(routes)/collection/page.tsx

```diff
@@ -1,61 +1,59 @@
-"use client"
+"use client";
 import Cardcontainer from "@/app/component/common/Cards/cardcontainer";
 import feedsFinder from "@/app/component/functions/findFees";
 import { FC, useState } from "react";
-import { Address, useAccount, useBlockNumber} from "wagmi";
+import { Address, useAccount, useBlockNumber } from "wagmi";
 import Balance from "@/app/component/functions/BalanceFinder";
 import getBlockTimeStamp from "@/app/component/functions/getBlocktimeStamp";
 import { LayoutRouterContext } from "next/dist/shared/lib/app-router-context.shared-runtime";
 import { parse } from "path";
 
-interface pageProps {
-    
-}
- 
-const page: FC<pageProps> = () => {
-  const [selectedValue, setSelectedValue] = useState<string>('1');
+interface pageProps {}
+
+const Page: FC<pageProps> = () => {
+  const [selectedValue, setSelectedValue] = useState<string>("1");
   const [showResult, setShowResult] = useState<boolean>(false);
-    const {data, isError, isLoading} = useBlockNumber()
-   
-    const handleGetBlockTime = (s: string)=>{
-      const n = parseInt(s);
-      if(isLoading){
-         return <div>Fetching block number</div>
-      }
-      if(isError){
-         return <div>Error Fetching block number</div>
-      }
-           let amountAdd =  216000 * n//216000 is the number of blocks that represents 30 days
-      let loanRepayDeadLineAddedAmount = BigInt(amountAdd);
-      let loanRepayDeadLine = loanRepayDeadLineAddedAmount;
-      if(data?.toString().length){  
-        loanRepayDeadLine = loanRepayDeadLine + data
-        return <div> the date in blocks is {loanRepayDeadLine.toString()}</div>
-      } 
-      
-      return <div>none</div>
-     
-    } 
-   
-  const { address} = useAccount()
+  const { data, isError, isLoading } = useBlockNumber();
+
+  const handleGetBlockTime = (s: string) => {
+    const n = parseInt(s);
+    if (isLoading) {
+      return <div>Fetching block number</div>;
+    }
+    if (isError) {
+      return <div>Error Fetching block number</div>;
+    }
+    let amountAdd = 216000 * n; //216000 is the number of blocks that represents 30 days
+    let loanRepayDeadLineAddedAmount = BigInt(amountAdd);
+    let loanRepayDeadLine = loanRepayDeadLineAddedAmount;
+    if (data?.toString().length) {
```

Comment:
Lets not use `let` use`const` it self

When you are using let you are telling the reader the variable is going to be modified down the code again which makes hard to follow up

Reply:
this code was written by @Greyastratos ,
But I will make the amendment